lib/bolt/target.rb

```diff
@@ -157,9 +157,39 @@ class Target
 
     # Satisfies the Puppet datatypes API
     def self.from_asserted_hash(hash)
+      inv = Puppet.lookup(:bolt_inventory)
```

Comment:
We should be able to instantiate a logger here and use that instead of having to pull in the inventory.
```
logger = Logging.logger[self]
...
logger.warn("...")
```